File: src/test/java/testee/it/reportng/HTMLToBase64Test.java
package testee.it.reportng;

import org.testng.annotations.Test;

import java.io.IOException;

public class HTMLToBase64Test {
    @Test
    public void testHTMLToBase64() throws IOException {
        assert HTMLToBase64.htmlToBase64(
                HTMLToBase64.html, 1024, 768, "e2e.png").length() > 0 :
                "Wrong html to base64 encoding";
    }
}

File: src/test/kotlin/testee/it/reportng/HTMLToBase64Test.kt
package testee.it.reportng

import org.testng.annotations.Test
import java.io.IOException

class HTMLToBase64Test {

    @Test
    @Throws(IOException::class)
    fun testHTMLToBase64() {
        assert(HTMLToBase64.htmlToBase64(
                HTMLToBase64.html, 1024, 768, "e2e.png").isNotEmpty()) { "Wrong html to base64 encoding" }
    }
}